Modify CmakeLists.txt to install libfaiss_gpu.a.

Copyright (c) 2024 Di-Is

This software is released under the MIT License.
http://opensource.org/licenses/mit-license.php

diff --git a/faiss/faiss/gpu/CMakeLists.txt b/faiss/faiss/gpu/CMakeLists.txt
index 126cbe50..0daf907d 100644
--- a/faiss/faiss/gpu/CMakeLists.txt
+++ b/faiss/faiss/gpu/CMakeLists.txt
@@ -250,6 +250,7 @@ if(FAISS_ENABLE_RAFT)
 endif()
 
 add_library(faiss_gpu STATIC ${FAISS_GPU_SRC})
+install(TARGETS faiss_gpu ARCHIVE DESTINATION lib)
 set_target_properties(faiss_gpu PROPERTIES
   POSITION_INDEPENDENT_CODE ON
   WINDOWS_EXPORT_ALL_SYMBOLS ON
